apps/src/music/views/MusicView.jsx

```diff
@@ -764,6 +764,7 @@ class UnconnectedMusicView extends React.Component {
         />
         <MusicLabView
           blocklyDivId={BLOCKLY_DIV_ID}
+          isPlaying={this.props.isPlaying}
```

Comment:
I believe `isPlaying` and `lastMeasure` are also stored in redux, so we don't even need to send them via props here. And I think we already provide a `setPlaying` function as a prop, which `MusicLabView` can use to stop the song by calling `setPlaying(false)`.

Reply:
Now using `isPlaying` and `lastMeasure` from redux.  [`stopSong`](https://github.com/code-dot-org/code-dot-org/blob/4a59f1fe75b91f7b99ffc1d09d74bdd03a4420c4/apps/src/music/views/MusicView.jsx#L718-L741) seems to do a number of extra things, which I presume we'd still want to do.

Reply:
Great! Yep, `setPlaying(false)` should call `stopSong` I believe, as per: https://github.com/code-dot-org/code-dot-org/blob/4a59f1fe75b91f7b99ffc1d09d74bdd03a4420c4/apps/src/music/views/MusicView.jsx#L551-L558
There's a probably a better name we could use for this function, so it doesn't get confused with the similarly named redux action `setIsPlaying` 🙂

Reply:
Ah, I think that was my confusion.  Updated.